core-mysql/src/main/resources/reference.conf

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: Apache-2.0
+
+// #connection-settings
+pekko.persistence.r2dbc {
+
+  # setting dialect to empty toggles use of the following class config
```

Comment:
The values allowed are empty and "mysql" ? Could we list the allowed values in the comment?

Reply:
No, valid `dialect` values are declared here - https://github.com/apache/pekko-persistence-r2dbc/blob/main/core/src/main/resources/reference.conf#L105. Setting `dialect` to empty enables loading of DAO classes specified by `journalDaoClass`, `queryDaoClass`, `snapshotDaoClass`, `durableStateDaoClass` configs instead of using default Postgres/Yugabyte implementation. This is the only way I figured MySQL implementation could be plugged in as a separate sbt module. This also provides flexibility in a way that library's users could plug their own DAO implementations and provide support for new databases without making changes to the library.

If you have any suggestions for improvements to implementation or docs, please let me know,

Reply:
I am not sure how else to explain it without you having to familiarize with the code more to either understand my solution or propose an alternative that is better.